mep/people/tests/test_admin.py

```diff
@@ -1,22 +1,39 @@
 from unittest.mock import Mock, patch
-
+from io import StringIO
+import csv
+import os
+import random
+import tempfile
 from django.contrib import admin
-from datetime import date
-
+from datetime import date, datetime
 from django.http import HttpResponseRedirect
-from django.test import TestCase
+from django.test import TestCase, Client
 from django.urls import reverse
 from django.utils.timezone import now
+from django.apps import apps
 
 from mep.accounts.models import Account, Subscription
 from mep.books.models import Creator, CreatorType, Work
-from mep.people.admin import PersonAdmin, PersonTypeListFilter
-from mep.people.models import Person, PastPersonSlug
-
+from mep.people.admin import PersonAdmin, PersonTypeListFilter, PersonAdminImportExport, PERSON_IMPORT_EXPORT_COLUMNS, ExportPersonResource
+from mep.people.models import Person, PastPersonSlug, Country
+from django.conf import settings
 
 class TestPersonAdmin(TestCase):
     fixtures = ["sample_people"]
 
+    def setUp(self):
+        User = apps.get_model("auth", "User")
+        # script user needed for log entry logic
+        # store the password to login later
+        password = 'adminpass' 
+        self.admin_user = User.objects.create_superuser('admin', 'admin@admin.com', password)
```

Comment:
I've had code like this flagged for including passwords in source code (even though it's test code). Was looking elsewhere to see what we've done in this code base - in newer projects we're using pytest-django with an admin client fixture but I don't think that's easy to use here.

One workaround (used elsewhere in mep-django) has been to generate a password so it isn't a hard-coded string. (You'll have to import uuid for this to work)

```suggestion
        password = str(uuid.uuid4())
        self.admin_user = User.objects.create_superuser('admin', 'admin@admin.com', password)
```